barretenberg/cpp/src/barretenberg/chonk/chonk.cpp

```diff
@@ -721,20 +725,61 @@ Chonk::VerificationKey Chonk::get_vk() const
 void Chonk::update_native_verifier_accumulator(const VerifierInputs& queue_entry,
                                                const std::shared_ptr<Transcript>& verifier_transcript)
 {
+    info("-------- DEBUGGING INFO FOR NATIVE FOLDING STEP --------");
+
     auto verifier_inst = std::make_shared<VerifierInstance>(queue_entry.honk_vk);
 
     FoldingVerifier native_verifier(verifier_transcript);
     if (queue_entry.type == QUEUE_TYPE::OINK) {
-        auto [_, new_accumulator] = native_verifier.instance_to_accumulator(verifier_inst, queue_entry.proof);
+        auto [_first_verified, new_accumulator] =
+            native_verifier.instance_to_accumulator(verifier_inst, queue_entry.proof);
         native_verifier_accum = std::move(new_accumulator);
+
+        info("Was the Sumcheck turning an instance into an accumulator verified? ", _first_verified ? "true" : "false");
```

Comment:
Nit: all of these pose a question that isn't so naturally answered with true/false :) - maybe something like 
```suggestion
        info("Sumcheck: instance to accumulator verified: ", _first_verified ? "true" : "false");
```